packages/react-auth/src/AuthProvider/index.tsx

```diff
@@ -246,7 +258,23 @@ export const AuthProvider: FC<AuthProviderProps> = ({
       setError(msg);
       setLoading(false);
     } else {
-      axios
+      const { success, response } = await auth.post(infoURL, { code: code });
```

Comment:
should the client just have the client endpoints as methods, so one for client/userinfo

Reply:
yes, definitely should be a method. added a `userinfo` method on the client